bundles/rke2/uds-bundle.yaml

```
@@ -33,7 +33,7 @@ packages:
   # Zarf init
   - name: init
     repository: ghcr.io/zarf-dev/packages/init
-    ref: v0.39.0
+    ref: v0.41.0
```

```suggestion
    ref: v0.40.1
```
This needs to match the version of zarf that the version of UDS we are requiring is using.
uds 0.16.0 = zarf 0.39.0
uds 0.17.0 = zarf 0.40.1